diesel/src/connection/mod.rs

```diff
@@ -97,4 +97,6 @@ pub trait Connection: SimpleConnection + Sized {
     #[doc(hidden)] fn rollback_transaction(&self) -> QueryResult<()>;
     #[doc(hidden)] fn commit_transaction(&self) -> QueryResult<()>;
     #[doc(hidden)] fn get_transaction_depth(&self) -> i32;
+    #[doc(hidden)] fn manage_updated_at_for_table(&self, &str) -> QueryResult<()>;
+    #[doc(hidden)] fn neglect_updated_at_for_table(&self, &str) -> QueryResult<()>;
```

Comment:
It's probably not the most self-explanatory name if it doesn't appear next to the `manage` form. I'm actually not sure why these are Rust functions. Hopefully it'll become clear as I read on.